feature/err/aggregate/index.js

```diff
@@ -137,6 +137,7 @@ function canonicalizeStackURLs (stackInfo) {
 }
 
 function storeError (err, time, internal, customAttributes) {
+  // console.log('storeError', err)
```

Comment:
This and the other console.log needs to be removed?

Reply:
Oops, nvm, this is old stuff.

Reply:
Good catch, removed from this file